build/azDevOps/azure/azuredevops-vars.yml

```diff
@@ -90,7 +90,7 @@ variables:
   - name: maven_package_version
     ${{ if eq( variables['Build.SourceBranchName'], 'main' ) }}:
       value: "${{ variables.version_major }}.${{ variables.version_minor }}.${{ variables.version_patch }}"
-    ${{ if ne( variables['Build.SourceBranchName'], 'main' ) }}:
+    ${{ if ne( variables['Build.SourceBranchName'], 'dev-change' ) }}:
```

Comment:
Why this?

Reply:
it just a quick change to test a branch, i rolled back

Reply:
I'll leave the comment here till then, so we won't forget